tensorflow_lite_support/c/task/processor/BUILD

```diff
@@ -0,0 +1,14 @@
+load("@org_tensorflow//tensorflow/lite/core/shims:cc_library_with_tflite.bzl", "cc_library_with_tflite")
+
+package(
+    default_visibility = ["//tensorflow_lite_support:users"],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+cc_library(
+    name = "classification_result_c_api",
```

Comment:
As we now have a dedicated `c` folder, let's remove the suffix `c_api` from the target and file names.

Reply:
Changed file names and associated build scripts. Have not removed the suffix in the names in the rule definitions for libraries in Bazel to avoid any confusion.